vnext/Microsoft.ReactNative/Views/Image/ReactImage.cpp

```diff
@@ -277,7 +285,7 @@ winrt::fire_and_forget ReactImage::SetBackground(bool fireLoadEndEvent) {
                 winrt::LoadedImageSurface const & /*sender*/,
                 winrt::LoadedImageSourceLoadCompletedEventArgs const &args) {
               if (auto strong_this{weak_this.get()}) {
-                bool succeeded{false};
+                std::wstring error{L""};
```

Comment:
>L"" [](http://example.com/codeflow?start=35&length=3)

Nit: Isn't this what the default ctor does?

Reply:
yes this should not be necessary